packages/utils/src/lib/reports/log-stdout-summary.integration.test.ts

```diff
@@ -0,0 +1,47 @@
+import { beforeAll, describe, expect, vi } from 'vitest';
+import { reportMock } from '@code-pushup/test-utils';
+import { ui } from '../logging';
+import { logStdoutSummary } from './log-stdout-summary';
+import { scoreReport } from './scoring';
+import { sortReport } from './sorting';
+
+describe('logStdoutSummary', () => {
+  // eslint-disable-next-line vitest/require-hook
+  let logs: string[] = [];
+  beforeAll(() => {
```

Comment:
I think it's enough to move the initialization into a hook to satisfy the lint rule.

```suggestion
  let logs: string[];
  beforeAll(() => {
    logs = [];
```